discord/utils.py

```diff
@@ -108,7 +108,7 @@
 )
 
 DISCORD_EPOCH = 1420070400000
-DEFAULT_FILE_SIZE_LIMIT_BYTES = 26214400
+DEFAULT_FILE_SIZE_LIMIT_BYTES = 10000000
```

Comment:
```suggestion
DEFAULT_FILE_SIZE_LIMIT_BYTES = 10485760
```

Reply:
huh. I just went with the amount as advertised (which uses `MB`), guess it's not really a problem for them to under-promise slightly.

Reply:
![image](https://github.com/user-attachments/assets/e4b42617-35ab-4528-96eb-78f177809cb6)